src/test/ProposalPayloadE2E.t.sol

```diff
@@ -24,7 +24,7 @@ contract ProposalPayloadE2ETest is Test {
     address public constant VARIABLE_DEBT_FEI_WHALE = 0x26bdDe6506bd32bD7B5Cc5C73cd252807fF18568;
 
     function setUp() public {
-        vm.createSelectFork(vm.rpcUrl("mainnet"));
+        vm.createSelectFork(vm.rpcUrl("mainnet"), 15745470);
```

Comment:
as you're working with addresses like `VARIABLE_DEBT_FEI_WHALE` which were/are a whale at a specific point in time i think it makes sense to also fix the block in the test.

Also not having a specific block makes tests painfully slow.